src/CONST.ts

```diff
@@ -3607,6 +3609,272 @@ const CONST = {
+                        '3. Click <strong>Members</strong> > <strong>Invite member</strong>.\n' +
+                        '4. Enter emails or phone numbers. \n' +
+                        '5. Add an invite message if you want.\n' +
+                        '\n' +
+                        'That’s it! Happy expensing :)',
+                },
+            ],
+        },
+        [onboardingChoices.PERSONAL_SPEND]: {
+            message: 'Here’s how to track your spend in a few clicks.',
+            video: {
+                url: `${CLOUDFRONT_URL}/videos/intro-1280.mp4`,
+                thumbnailUrl: `${CLOUDFRONT_URL}/images/expensify__favicon.png`,
+                duration: 55,
+                width: 1280,
+                height: 960,
+            },
+            tasks: [
+                {
+                    type: 'trackExpense',
+                    autoCompleted: false,
+                    title: 'Track an expense',
+                    subtitle: 'Track an expense in any currency, whether you have a receipt or not.',
+                    message:
+                        'Here’s how to track an expense:\n' +
+                        '\n' +
+                        '1. Click the green + button.\n' +
+                        '2. Choose <strong>Track expense</strong>.\n' +
+                        '3. Enter an amount or scan a receipt.\n' +
+                        '4. Click Track.\n' +
+                        '\n' +
+                        'And you’re done! Yep, it’s that easy.',
+                },
+            ],
+        },
+        [onboardingChoices.CHAT_SPLIT]: {
+            message: 'Splitting bills with friends is as easy as sending a message. Here’s how.',
+            video: {
+                url: `${CLOUDFRONT_URL}/videos/intro-1280.mp4`,
+                thumbnailUrl: `${CLOUDFRONT_URL}/images/expensify__favicon.png`,
+                duration: 55,
+                width: 1280,
+                height: 960,
+            },
+            tasks: [
+                {
+                    type: 'startChat',
+                    autoCompleted: false,
+                    title: 'Start a chat',
+                    subtitle: 'Start a chat with a friend or group using their email or phone number.',
+                    message:
+                        'Here’s how to start a chat:\n' +
+                        '\n' +
+                        '1. Click the green + button.\n' +
+                        '2. Choose <strong>Start chat</strong>.\n' +
+                        '3. Enter emails or phone numbers.\n' +
+                        '\n' +
+                        'If any of your friends aren’t using Expensify already, they’ll be invited automatically. \n' +
+                        '\n' +
+                        'Every chat will also turn into an email or text that they can respond to directly.',
+                },
+                {
+                    type: 'splitExpense',
```

Comment:
Seems like this `type: 'splitExpense'` is not supported in the backend https://github.com/Expensify/Auth/blob/7c6c671e0b1e2886d5f46393a37a32bed6a0609a/auth/lib/GuidedSetup.cpp#L12-L21 and is throwing:

https://www.expensify.com/_devportal/tools/logSearch/#query=request_id:(%2287873a9bdcf5e6ba-HKG%22)+AND+timestamp:[2024-04-22T16:10:43.954Z+TO+2024-04-22T18:10:43.954Z]&index=logs_expensify-027961

Reply:
cc @francoisl @mountiny

Reply:
Yep known issue, there's a backend PR in progress for this - https://github.com/Expensify/Auth/pull/10606.